### DISCUSSION TITLE
Notifications outside HA

### DISCUSSION THREAD

**Discussion by maylorian** [blakeblackshear/frigate#3366] [Ask A Question] [answered]
Hello there!

I couldn't find anything related to this in the docs, so I thought I would ask here. Does the functionality exist to send direct notifications to any chat messenger (slack, telegram etc)? I know this is not what Frigate exists to do but, I'm not currently running a HA instance and I would really love to get notifications along the nvr functionality.

If not, I was planning on creating one myself. Is the following process right, or is there a simpler way?

1. Listen to frigate/events over MQTT for new detections
2. Make a call to the relevant API endpoint to retreive the actual image
3. Send the notification to the messenger of choice.

Many thanks for the excellent product you provide!

**Comment by NickM-27** [comment, marked as answer]
Frigate itself does not provide this functionality. But with MQTT this functionality can easily be created. It could be done in NodeRED, probably some other services as well. But something lightweight and purpose-built for this could make things easier for sure.

**Comment by adriel** [comment]
Did anyone end up doing any work on this? I'm kinda lost when it comes to MQTT. I just want to feed notifications to Discord.

**Comment by squiddity** [comment]
Just seeing this post a year later but yes I happily feed notifications to Discord using a small script at https://github.com/squiddity/nvr/blob/master/mqtt-discord/bot.py. It just relies on two environment variables, DISCORD to contain your discord bot API key, and MQTT the url of your mqtt server. It doesn't support username/password for MQTT but you could add that. Finally, it assumes that in the discord server there exists a channel of the same name as the camera within frigate, and simply posts the snapshot pulled from MQTT onto that channel.
